README.md

```diff
@@ -229,13 +229,13 @@ Add the dependencies to your project:
 
 ```groovy
 dependencies {
```

Comment:
I'm pretty sure this should now be:

```groovy
dependencies {
    api 'io.jsonwebtoken:jjwt-api:0.10.2'
    runtimeOnly 'io.jsonwebtoken:jjwt-impl:0.10.2' 
    runtimeOnly('io.jsonwebtoken:jjwt-orgjson:0.10.2') {
        exclude group: 'org.json', module: 'json' //provided by Android natively
    }
    // Uncomment the next line if you want to use RSASSA-PSS (PS256, PS384, PS512) algorithms:
    //runtimeOnly 'org.bouncycastle:bcprov-jdk15on:1.60'
}
```

and proguard additions should be:

```
-dontwarn org.json.JSONString
-dontwarn org.json.JSONWriter```

Could you please confirm?

Reply:
Yes that works.

Reply:
Thanks!

Reply:
I've updated this in the docs accordingly.  I'll get this out in the 0.10.3 release.  Thanks for your help!